src/OrchardCore/OrchardCore.Abstractions/BackgroundTasks/BackgroundTaskExtensions.cs

```diff
@@ -12,41 +12,30 @@ public static BackgroundTaskSettings GetDefaultSettings(this IBackgroundTask tas
             var technicalName = task.GetTaskName();
 
             var attribute = task.GetType().GetCustomAttribute<BackgroundTaskAttribute>();
-
             if (attribute != null)
             {
                 return new BackgroundTaskSettings
                 {
-                    Title = !String.IsNullOrWhiteSpace(attribute.Title) ? attribute.Title : technicalName,
                     Name = technicalName,
+                    Title = !String.IsNullOrWhiteSpace(attribute.Title) ? attribute.Title : technicalName,
                     Enable = attribute.Enable,
                     Schedule = attribute.Schedule,
                     Description = attribute.Description,
                     LockTimeout = attribute.LockTimeout,
-                    LockExpiration = attribute.LockExpiration
+                    LockExpiration = attribute.LockExpiration,
                 };
             }
 
             return new BackgroundTaskSettings()
             {
                 Name = technicalName,
-                Title = technicalName
+                Title = technicalName,
```

Comment:
Still I'm wondering why you adding a comma at the end :)

Reply:
This is an analyzer rule we are using in some projects, for example it is easier to re-order the lines without dealing with the commas, even if we move the last line, I like it.

https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1413.md